### DISCUSSION TITLE
What if there was SSF (Scalable Screen Font) support?

### DISCUSSION THREAD

**Discussion by Neil-Ciciglycerin** [CosmosOS/Cosmos#2932] [Ideas]
This would allow you to easily scale text, as well as giving you the ability to have colors in your characters. It also has a friendly open source library!

**Comment by zarlo** [comment]
are you talking about this? https://wiki.osdev.org/Scalable_Screen_Font

**Comment by Neil-Ciciglycerin** [comment]
Yep!
